src/net/sourceforge/kolmafia/request/StillSuitRequest.java

```diff
@@ -52,7 +52,7 @@ public static boolean canMake() {
       return false;
     }
 
-    return true;
+    return DrinkItemRequest.allowBoozeConsumption("stillsuit distillate", 1);
```

Comment:
I don't know if it's right to have a user prompt here since canMake is used in Concoction.canMake

Reply:
Should that move to `run()`, then?

Reply:
I knew at the time but have forgotten now. Maybe!